### PR TITLE
(MODULES-2783) Missing ip6tables service name

### PR DESCRIPTION
**Pull request by abednarik**

### CODE REVIEW

**Comment by pshaikh** [comment]
We use both iptables & ip6tables, for this we set service_name of firewall class as ['iptables','ip6tables'], the change though is excellent, will it work in situation where one passes service_name as above?

**Comment by abednarik** [comment]
I think the way you were using this module did-t work since they have iptables service hardcoded.

**Comment by pshaikh** [comment]
You're right...iptables was hardcoded, however how to enable both iptables & ip6tables on CentOS or RedHat? By default its always iptables. So what I am doing is passing ['iptables','ip6tables'] for "service_name" attribute of class firewall. This is also done for Gentoo linux. In such a case this may not work.

File: manifests/linux/redhat.pp [hunk 1]
```diff
@@ -46,7 +46,7 @@
       exec { '/usr/bin/systemctl daemon-reload':
         require => Package[$package_name],
         before  => Service[$service_name],
-        unless  => '/usr/bin/systemctl is-active iptables'
+        unless  => '/usr/bin/systemctl is-active iptables',
```
Comment (DavidS) [inline manifests/linux/redhat.pp]: shouldn't this also use $service_name?

**Comment by abednarik** [comment]
Thanks @DavidS for providing feedback. Will do that today.

Cheers.

**Comment by abednarik** [comment]
Done @DavidS 

 Any feedback is welcome.

**Comment by DavidS** [comment]
I'll merge it as is. For @pshaikh 's use-case (and I guess this will become more necessary in the coming time) I'd really love to see explicit IPv6 support alongside the current types, so that it can be selected properly at the top-level.

**Pull request merged**